### ISSUE TITLE
[containerized] jira issue view throws "Error: exit status 1"

### ISSUE THREAD

**Issue by edsantiago** [ankitpokhrel/jira-cli#498]
**Describe the bug**
```console
$ podman [...] run --rm -it --entrypoint /bin/jira ghcr.io/ankitpokhrel/jira-cli:latest issue view RUN-1477
Error: exit status 1
```

**Please provide following details**
1. JiraCLI Version:
```
(Version="v1.1.0", GitCommit="a88eb61e142a6e604dc1f9f30884fff618277d22", CommitDate="2022-08-14T08:43:18+00:00", GoVersion="go1.17.11", Compiler="gc", Platform="linux/amd64")
```
Also tried building locally (same results):
```
(Version="main", GitCommit="997ead6cfaad335563b22b8b1fee69bcad4a91c3", CommitDate="2022-11-14T14:35:44+00:00", GoVersion="go1.17.11", Compiler="gc", Platform="linux/amd64")
```
3. Are you using Jira cloud or on-premise jira server? Also mention the version for on-premise installation.
```
on-premise, Jira v8.20.11
```
4. What operating system are you using? Also mention version.
```
Fedora 36 up-to-date
```
5. What terminal are you using? Also mention version.
```
rxvt-unicode-9.30-2.fc36.x86_64
lxterminal-0.4.0-4.fc36.x86_64
xterm-371-1.fc36.x86_64
and, over ssh, xfce4-terminal-1.0.4
```

**To Reproduce**
```console
$ podman [...]  issue view --debug  RUN-1477
Using config file: /root/.config/.jira/.config.yml
⠴ Fetching issue details...

REQUEST DETAILS
------------------------------------------------------------

GET /rest/api/2/issue/RUN-1477 HTTP/1.1
Host: issues.redhat.com
Authorization: Bearer [snip]



RESPONSE DETAILS
------------------------------------------------------------

HTTP/1.1 200 OK
Cache-Control: max-age=0, no-cache, no-store
Connection: keep-alive
Content-Security-Policy: sandbox
Content-Type: application/json;charset=UTF-8
Date: Mon, 14 Nov 2022 14:40:01 GMT
Expires: Mon, 14 Nov 2022 14:40:01 GMT
Pragma: no-cache
Referrer-Policy: strict-origin-when-cross-origin
Set-Cookie: JSESSIONID=x; Path=/; Secure; HttpOnly
Set-Cookie: atlassian.xsrf.token=x; Path=/; Secure; SameSite=None
Set-Cookie: x; path=/; HttpOnly; Secure; SameSite=None
Strict-Transport-Security: max-age=31536000
Vary: User-Agent
Vary: Accept-Encoding
X-Anodeid: rh1-jira-prod-1
X-Arequestid: 880x....
X-Asessionid: ....
X-Ausername: ....
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Rh-Edge-Cache-Status: Miss from child, Miss from parent
X-Rh-Edge-Reference-Id: .....
X-Rh-Edge-Request-Id: ...
X-Seraph-Loginreason: OK
X-Xss-Protection: 1; mode=block

Error: exit status 1
```

**Expected behavior**

Adding `--plain` shows what I expect (Story, In Progress, etc etc)

I'm new to this, so haven't done a full functionality test. `issue list` works fine, even interactively. Typing `v` on an issue pops up a text alert "Processing. Please wait..." for a second or less, with no other effect. I assume it's trying and failing to run `issue view`.

**Comment by edsantiago** [comment]
`jira board list` also fails with `Error: exit status 1` and no other output.

**Comment by ankitpokhrel** [comment]
Hi @edsantiago, I believe this will work with the following (something that needs to be fixed in the tool):

```sh
PAGER='less -R' jira issue view <key>
PAGER='less -R' jira board list
```

**Comment by edsantiago** [comment]
That did indeed resolve it! Thank you @ankitpokhrel, and thank you for this tool.

**Comment by ankitpokhrel** [comment]
This should be fixed with #503 and will be available in the next release.
